SheepDash/core/src/com/missionbit/game/sprites/Obstacles.java

```diff
@@ -0,0 +1,46 @@
+package com.missionbit.game.sprites;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.math.Vector2;
+
+import java.util.Random;
+
+/**
+ * Created by missionbit on 6/29/17.
+ */
+
+public class Obstacles {
```

Comment:
Classic java naming conventions might call this class Obstacle.  If the class contains a collection of obstacles, Obstacles would be appropriate.  It's a stylistic choice- not required.